### ISSUE TITLE
RCS1085: false positive when backing field has attribute

### ISSUE THREAD

**Issue by pyrocumulus** [JosefPihrt/Roslynator#870]
**Product and Version Used**: 
Microsoft Visual Studio Professional 2019 Version 16.11.7
Roslynator 2019 version 3.3.0 extension

**Steps to Reproduce**:
Define a property in the following way:
```csharp
[MyAttribute]
private bool _myProperty;

public bool MyProperty { get => _myProperty; set { _myProperty = value; } }
```

**Actual Behavior**:
Refactoring 'RCS1085: Use auto-implemented property' is suggested. Applying the refactoring results in the following new property:

```csharp
public bool MyProperty { get; set; }
```
But this is incorrect. Roslynator removes the attribute completely thus resulting in code that is not equivalent to the state it was before the refactoring.

Visual Studio also has this suggestion/refactoring ('IDE0032 - Use auto property') but that does work correctly. It will not suggest the refactoring when the attribute is there, it does when I remove it.

**Expected Behavior**:
Do not suggest RCS1085 when the backing field has an attribute assigned to it.
